go/client/getentries.go

```diff
@@ -13,20 +13,19 @@ import (
 	"golang.org/x/net/context"
 )
 
-// LeafEntry respresents a JSON leaf entry
+// LeafEntry respresents a JSON leaf entry.
 type LeafEntry struct {
 	LeafInput []byte `json:"leaf_input"`
 	ExtraData []byte `json:"extra_data"`
 }
 
-// getEntriesReponse respresents the JSON response to the CT get-entries method
-type getEntriesResponse struct {
+// GetEntriesResponse respresents the JSON response to the CT get-entries method.
+type GetEntriesResponse struct {
```

Comment:
I guess I was lucky with that, my only use was of a `entries, err := GetRawEntries(...)` form, so it worked anyway, even if I couldn't explicitly declare that `entries` variable!